### ISSUE TITLE
The app crashes every time I play music

### ISSUE THREAD

**Issue by GRbit** [VinylMusicPlayer/VinylMusicPlayer#1120]
The app is magnificent, I like it very much. But it started crashing every time I try to play anything.
In fact, it even play the music, but I always receive a pop up notification that the app has crashed with a choice to wait or to close it. 

I don't know what other information could be usefull, but I'm willing to help. 

App version: 1.11.0
App version code: 202
Android build version: eng.20240716.224107
Android release version: 13
Android SDK version: 33
Android build ID: e_beyond0lte-user 13 TQ3A.230901.001 eng.root.20240716.224107
Device brand: samsung
Device manufacturer: samsung
Device name: beyond0lte
Device model: SM-G970F
Device product name: lineage_beyond0lte
Device hardware name: exynos9820
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]

## Time: 
2024/08/23 13:26:35
## Stack: 
java.lang.IllegalStateException: Couldn't read row 794, col 0 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.
	at android.database.CursorWindow.nativeGetLong(Native Method)
	at android.database.CursorWindow.getLong(CursorWindow.java:539)
	at android.database.AbstractWindowedCursor.getLong(AbstractWindowedCursor.java:78)
	at com.poupa.vinylmusicplayer.provider.SongPlayCountStore.updateExistingRow(Unknown Source:6)
	at com.poupa.vinylmusicplayer.provider.SongPlayCountStore.updateResults(Unknown Source:54)
	at com.poupa.vinylmusicplayer.provider.SongPlayCountStore.<init>(Unknown Source:18)
	at com.poupa.vinylmusicplayer.provider.SongPlayCountStore.getInstance(Unknown Source:13)
	at com.poupa.vinylmusicplayer.service.MusicService.handleChangeInternal(Unknown Source:109)
	at com.poupa.vinylmusicplayer.service.MusicService.handleAndSendChangeInternal(Unknown Source:0)
	at com.poupa.vinylmusicplayer.service.MusicService.notifyChange(Unknown Source:0)
	at com.poupa.vinylmusicplayer.service.MusicService.openTrackAndPrepareNextAt(Unknown Source:20)
	at com.poupa.vinylmusicplayer.service.MusicService.playSongAtImpl(Unknown Source:0)
	at com.poupa.vinylmusicplayer.service.PlaybackHandler.handleMessage(Unknown Source:177)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.os.HandlerThread.run(HandlerThread.java:67)

**Comment by GRbit** [comment]
I've spend some time googling on this error and found a stackoverflow post:
https://stackoverflow.com/questions/21432556/android-java-lang-illegalstateexception-couldnt-read-row-0-col-0-from-cursorw
TLDR: it says that this error is returned in case SQLite row is bigger than 2Mb. I have quite a lot of music on my device, more than 20k files.

I've checked out the app on my device, and it stated that it has 1,5Gb of user data. After deleting all the user data and cache the app started to work.

Not sure if this could be submitted as a bug, since it can be fixed so easily. I suppose it doesn't worth to spend time to fix it.
